volume-providers/vpc/provider/list_volumes.go

```diff
@@ -26,6 +27,16 @@ func (vpcs *VPCSession) ListVolumes(limit int, start string, tags map[string]str
 	defer vpcs.Logger.Info("Exit ListVolumes", zap.Reflect("start", start), zap.Reflect("filters", tags))
 	defer metrics.UpdateDurationFromStart(vpcs.Logger, "ListVolumes", time.Now())
 
+	if limit < 0 {
+		return nil, userError.GetUserError("InvalidListVolumesLimit", fmt.Errorf(
+			"listVolumes got invalid entries request %v, supports values between 0-100", limit))
```

Comment:
can you combine this error msg to `InvalidListVolumesLimit` so that it will be reviewed by doc team and it will be good to have at one place.

Reply:
Fixed.